front/style/mixins/_positioning.scss

```diff
@@ -0,0 +1,14 @@
+@mixin vertical--align {
+    top: 50%;
+    transform: translateY(-50%);
+}
+
+@mixin full--size {
+	width: 100%;
+	height: 100%;
+}
+
+@mixin full--screen {
+	width: 100vw;
+	height: 100vh;
+}
```

Comment:
missing newline at end of file

Reply:
Also should be 4 spaces instead of 8 spaces above.